cactusbot/handlers/command.py

```diff
@@ -89,6 +89,9 @@ def __init__(self, channel, api):
 
         json = await response.json()
 
+        if not json["data"]["attributes"].get("enabled"):
```

Comment:
Should be `.get("enabled", True)`, since we assume that a command is enabled.

With the current code, if `enabled` isn't a valid key, `.get` will return `None`, and `not None` will run the code in the `if` block.